metis-normalization/src/main/java/eu/europeana/normalization/normalizers/RemoveDuplicateStatementNormalizer.java

```diff
@@ -48,11 +55,9 @@ private static final XpathQuery getProxySubtagQuery(Namespace.Element subtag) {
   @Override
   public NormalizationReport normalize(Document edm) throws NormalizationException {
     final InternalNormalizationReport report = new InternalNormalizationReport();
-    final List<Element> elements = new ArrayList<>();
-    final Set<TextLanguagePair> foundPairs = new HashSet<>();
     for (XpathQuery[] fieldSet : FIELD_SETS_TO_EVALUATE) {
 
-      elements.clear();
```

Comment:
Why did you change that, does it cause a problem? I did this to fix the technical debt issues..(these issues are not visible on the IDE) :D
Some below

Reply:
I fixed it.